src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -2510,12 +2510,12 @@ namespace HeatBalanceManager {
 
             GasType = state.dataMaterial->Material(MaterNum).GasType(1);
             if (GasType >= 1 && GasType <= 4) {
-                state.dataMaterial->Material(MaterNum).GasWght(1) = GasWght(GasType);
-                state.dataMaterial->Material(MaterNum).GasSpecHeatRatio(1) = GasSpecificHeatRatio(GasType);
+                state.dataMaterial->Material(MaterNum).GasWght(1) = GasWght[GasType - 1];
+                state.dataMaterial->Material(MaterNum).GasSpecHeatRatio(1) = GasSpecificHeatRatio[GasType - 1];
```

Comment:
I tried to minimize adding `- 1` to all of these index lookups where possible. In some places, though, it just wasn't feasible to do the required changes to make that go away.